src/components/Slides.vue

```diff
@@ -141,9 +144,28 @@ export default {
       ]
     }
   },
-  methods: {
-    searchForSlide () {
-      console.log('Pressed search')
+  computed: {
+    filteredSlides () {
+      // check if something is typed into the search bar
+      if (this.searchString) {
+        // if there is something the in the search bar then filter the array fo current slides
+        let vm = this
```

Comment:
`vm` stands for 'virtual machine'?

Reply:
View Model actually

Reply:
Can you change the variable name to `view_model` then, please?